src/lib/API/api.js

```diff
@@ -204,7 +204,60 @@ class API {
         return r
       })
   }
+
+  /**
+   * `/w3Site/api/wl/user` get user from web3 by token
+   * @param {string} token
+   */
+  getUserFromW3ByToken(token: string): Promise<$AxiosXHR<any>> {
+    let instance: AxiosInstance = axios.create({
```

Comment:
You are creating the axios instance twice.
Either create the instance once with the baseurl or use the client we already have but change its parameters.